src/scoreCard.js

```diff
@@ -1,29 +1,18 @@
+import createCard  from './createCard';
```

Comment:
Oh I put that in there when i was trying to figure out why it wasn't working, thought maybe I need to import it... nope. I'll take it out and re-commit